### ISSUE TITLE
Libertranslate bug in Docker - "Container not installed in background"

### ISSUE THREAD

**Issue by alexeyrosspage** [LibreTranslate/LibreTranslate#718]
If you try to install libertranslate with the -d flag, an error occurs.

`docker run --name libretranslate -d -p 5050:5000 libretranslate/libretranslate:latest`

However, if you install libertranslate with the -rm flag (subsequent deletion), the container runs until Docker closes. At first I thought it was a problem in Docker, but it turned out to be a bug in the container settings.

`docker run --name libretranslate -rm -p 5050:5000 libretranslate/libretranslate:latest`

**Comment by pierotofy** [comment]
Yes, seems the same as #712. This is due to the container "booting" (loading the language models can take a while). The app is unreachable until models have downloaded.

**Comment by pierotofy** [comment]
Should be fixed with latest version 🙏
